File: StrategyJava/src/br/work/QuickSort.java
package br.work;

import java.util.Collections;
import java.util.List;

public class QuickSort implements ISortStrategy {

	@Override
	public void Sort(List<String> list) {
		Collections.sort(list);// Default is Quicksort
  		 System.out.println("QuickSorted list");
		
	}

}

File: StrategyC#/Strategy/QuickSort.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Strategy
{
   public class QuickSort: SortStrategy
    {
        public override void Sort(List<string> list)
        {
            list.Sort(); // Default is Quicksort
            Console.WriteLine("QuickSorted list ");
        }
    }
}
